src/convertToObject.js

```diff
@@ -11,8 +12,19 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObj = sourceString.split(';')
+    .map(source => source.trim())
```

Comment:
It will be better to place all array operations on separate lines 
```suggestion
  const sourceObj = sourceString
    .split(';')
    .map(source => source.trim())
```